docs/tutorial/test_controller.rst

```diff
@@ -17,4 +20,8 @@ i.e. the control error:
 .. image:: ../pictures/ControllerTest2.jpg
 
 Feel free to experiment with the settings and see, 
-if the control loop reacts the way you would have predicted.
\ No newline at end of file
+if the control loop reacts the way you would have predicted.
+
+Keep in mind that the controller is implemented static.
+The control law does not adapt to changes in the model parameters 
+due to the linearized model used to calculate the controller gain.
```

Comment:
Using "default" parameters which may differ from the "real" parameters of the model when calculating the feedback is one thing, using a linearized model which is only sufficiently accurate around a certain steady state is another issue.
I think that it's a good idea to mention these problems but the the comment in its current form causes more confusion than clarity.